### PR TITLE
CXP-1021: add loading screen on wizard's confirm

### PR DESCRIPTION
**Pull request by tseho**
**Description (for Contributor and Core Developer)**

https://user-images.githubusercontent.com/93598018/146564658-6160521a-800f-4792-822e-a2d39c9997da.mp4

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Comment by tseho** [comment]
GTM

File: src/Akeneo/Connectivity/Connection/front/src/connect/components/AppWizardWithSteps/useConfirmHandler.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import {useConfirmAuthorization} from '../../hooks/use-confirm-authorization';
+import {PermissionFormProvider} from '../../../shared/permission-form-registry';
+import {PermissionsByProviderKey} from '../../../model/Apps/permissions-by-provider-key';
+
+type Result = {
+    confirm: () => Promise<void>;
+    processing: boolean;
+};
+
+export const useConfirmHandler = (
```
Comment (willy-ahva) [inline src/Akeneo/Connectivity/Connection/front/src/connect/components/AppWizardWithSteps/useConfirmHandler.ts:13]: missing test ?

File: src/Akeneo/Connectivity/Connection/front/src/connect/hooks/use-confirm-handler.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        );
+
+        window.location.assign(redirectUrl);
+    }, [confirmAuthorization, notify, translate, providers, permissions, notifyPermissionProviderError]);
+
+    return {
+        confirm: handleConfirm,
+        processing: processing,
+    };
+};
```
Comment (masacc) [inline src/Akeneo/Connectivity/Connection/front/src/connect/hooks/use-confirm-handler.ts:78]: is a test file missing for this hook ?

**Review by masacc — APPROVED** [review]

File: src/Akeneo/Connectivity/Connection/front/src/connect/hooks/use-confirm-handler.ts (hunk 2 above)
Comment (masacc) [reply src/Akeneo/Connectivity/Connection/front/src/connect/hooks/use-confirm-handler.ts:78]: ah just saw the comment of willy, same here, I resolve the conversation

**Comment by masacc** [comment]
Food for thoughts: is it something that we could/should reproduce on the permissions form in connected app settings page?

**Review by guillaumefer-akeneo — APPROVED** [review]

**Pull request merged**